### DISCUSSION TITLE
Count mount my config.yml file

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [Lissy93/dashy#1302] [Help and Support] [answered]
I am trying to deploy the following stack in Portainer:

**---
version: "3.8"
services:
  dashy:
    # To build from source, replace 'image: lissy93/dashy' with 'build: .'
    # build: .
    image: lissy93/dashy
    container_name: Dashy
    # Pass in your config file below, by specifying the path on your host machine
    #volumes:
    #  - ~/dashy/my-conf.yml:/app/public/conf.yml
    ports:
      - 4000:80
    # Set any environmental variables
    environment:
      - NODE_ENV=production
    # Specify your user ID and group ID. You can find this by running `id -u` and `id -g`
    #  - UID=1000
    #  - GID=1000
    # Specify restart policy
    restart: unless-stopped
    # Configure healthchecks
    healthcheck:
      test: ['CMD', 'node', '/app/services/healthcheck']
      interval: 1m30s
      timeout: 10s
      retries: 3
      start_period: 40s**

But I get the following error:
**failed to deploy a stack: Network my_dashy_default
Creating Network my_dashy_default
Created Container Dashy
Creating Container Dashy
Created Container Dashy
Starting Error response from daemon:
failed to create shim task:
OCI runtime create failed:
runc create failed:
unable to start container process:
error during container init:
error mounting “/dashy/my-conf.yml” to rootfs at “/app/public/conf.yml”:
mount /dashy/my-conf.yml:/app/public/conf.yml (via /proc/self/fd/6), flags: 0x5000: not a directory:
unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type**

~/dashy/my-conf.yml does exist and I have another Portainer stack that reference a different folder in the same way, so I think the permissions are correct.

**Comment by ZuluPilgrim** [comment]
I don't begin to understand how this all works but if I change the volumes part as follows (incorrect conf file name on host), the instance starts, It;s obviously not correct but dashy starts up.
`volumes:
      - ~/dashy/my-conf.yml:/app/public/onf.yml`

**Comment by ZuluPilgrim** [comment, marked as answer]
I came right. Permissions on  ~/dashy/my-conf.yml were not correct

**Comment by CrazyWolf13** [comment]
> I don't begin to understand how this all works but if I change the volumes part as follows (incorrect conf file name on host), the instance starts, It;s obviously not correct but dashy starts up.
> `volumes:
>       - ~/dashy/my-conf.yml:/app/public/onf.yml`
> 

That is not correct -v /path/on/host:/path/on/container

You do the above thing to access the config from outside what you did you told docker to mount a file that is not existing so it probably skipped that step and now you don't have access to your config file.
